src/generator/codegen.jl

```diff
@@ -311,9 +311,10 @@ function _emit_getproperty_ptr!(body, root_cursor, cursor, options)
         offset = getOffsetOf(getCursorType(root_cursor), n)
         if isBitField(field_cursor)
             w = getFieldDeclBitWidth(field_cursor)
-            @assert w <= 32 # Bit fields should not be larger than int(32 bits)
-            d, r = divrem(offset, 32)
-            ex = :(f === $(QuoteNode(fsym)) && return (Ptr{$ty}(x + $(4d)), $r, $w))
+            iszero(w) && continue
+            d, r = divrem(offset, 8)
+            @assert r + w <= 128 "Cannot handle bitfield larger than 128 bits"
```

Comment:
Is this documented in the standard?

Reply:
I don't think so. It's just super rare for a c/c++ code to have a primitive type that is larger than 128 bits, so I thought it would be reasonable to be left unhandled until we find a real use case.

Reply:
The test only has 64-bit test cases. Do you have a specific need for the 128-bit support?

Reply:
No, I only need 64-bit support